### PR TITLE
Iteration 1 documentation

### PR DESCRIPTION
**Pull request by karinamzalez**
- add documentation for: 
  - tray.js / tray-test.js
  - world.js / world-test.js

### CODE REVIEW

File: immersive_pahlka_documentation/iteration_1_doc.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+#### What is the 'tray' ?
+
+* Our game consists of a child in a lunchroom attempting to catch food items falling from the top of the screen.  The child's means of catching said food is a lunch tray.  There are two categories of food: _healthy_ and _unhealthy_.  Every _healthy_ food item that the child catches should stack upon the tray; the _unhealthy_ food removes a _healthy_ item from the top of the stack i.e. _unhealthy_ food does not stack.  For this iteration we are creating the basic template for said tray.
+
+#### What is the 'world' ?
+* The world is essentially the canvas-- i.e. the platform upon which our game happens.
+
+#### What are we testing ?
+* This iteration consists of two tests:
+  * Does the test exist?
```
Comment (deborahleehamel) [inline immersive_pahlka_documentation/iteration_1_doc.md]: "test" >>> "tray"?

Comment (deborahleehamel) [reply immersive_pahlka_documentation/iteration_1_doc.md]: add more about both tray an world tests that are included in this iteration?

File: immersive_pahlka_documentation/iteration_1_doc.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  * Can the user move the tray left and right using the left and right arrow keys?
+
+#### What are we using for Testing ?
+* Mocha - reference lesson above
```
Comment (deborahleehamel) [inline immersive_pahlka_documentation/iteration_1_doc.md]: and chai?

File: immersive_pahlka_documentation/tray_documentation.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  * Can the user move the tray left and right using the left and right arrow keys?
+
+#### What are we using for Testing ?
+* Mocha - reference lesson above
+* Chai - Gives us assertions
+
+----
+## Code
+#### Important Notes on Implementation :
+_ For further additions to code, one might find it beneficial to install the atom package, jshint. It helps one remain constantly mindful of JavaScript's syntactical discrepancies._
```
Comment (chompasina) [inline immersive_pahlka_documentation/tray_documentation.md:30]: Good call!

File: immersive_pahlka_documentation/world_documentation.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+# Iteration 2 Documentation
+## Create World
+----
+### Lesson Plans Used
+  * For Testing
+    * intro to Mocha//Unit level Testing in JavaScript: [lessonPlan](https://github.com/turingschool-examples/gametime-testing-journey/)
```
Comment (chompasina) [inline immersive_pahlka_documentation/world_documentation.md:6]: Let's capitalize the i in Intro for all of these in the next commits.

File: lib/index.js [hunk 5]
```diff
@@ -1,18 +1,10 @@
+require('../style.scss');
 const canvas = document.getElementById('game');
 const context = canvas.getContext('2d');
 const World = require('./world');
 
 var world = new World(canvas.width, canvas.height);
```
Comment (chompasina) [inline lib/index.js:7]: Glad we realized this was redundant.

**Comment by chompasina** [comment]
Looks good to me. This documentation is going to be so helpful for Pahlka!

**Pull request merged**